### PR TITLE
added checksums as a dependancy on line 101

### PR DESCRIPTION
**Pull request by ilevyor**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -98,6 +98,7 @@ aws_prepare_symbol_visibility_args(${PROJECT_NAME} "AWS_CAL")
 aws_add_sanitizers(${PROJECT_NAME} BLACKLIST "sanitizer-blacklist.txt")
 
 aws_use_package(aws-c-common)
+aws_use_package(aws-checksums)
```
Comment (justinboswell) [inline CMakeLists.txt:101]: You also need to add this as an upstream in `builder.json`

Comment (justinboswell) [reply CMakeLists.txt:101]: This will fix CI failures

Comment (ilevyor) [reply CMakeLists.txt:101]: resolved in patch2.

**Comment by ilevyor** [comment]
redoing correctly from console

**Pull request closed without merging**